src/Indicator.vala

```diff
@@ -18,15 +18,31 @@
 public class BluetoothIndicator.Indicator : Wingpanel.Indicator {
     public bool is_in_session { get; construct; default = false; }
 
-    private Widgets.PopoverWidget popover_widget;
-    private Widgets.DisplayWidget dynamic_icon;
+    public signal void device_requested (BluetoothIndicator.Services.Device device);
+    private BluetoothIndicator.Services.Device paired_device;
+    BluetoothIndicator.Widgets.PopoverWidget popover_widget;
+    Widgets.DisplayWidget? display_widget;
     private Services.ObjectManager object_manager;
 
     public Indicator (bool is_in_session) {
         Object (
             code_name: Wingpanel.Indicator.BLUETOOTH,
             is_in_session: is_in_session
         );
+
+        var settings = new Settings ("io.elementary.desktop.wingpanel.bluetooth");
+        display_widget = new Widgets.DisplayWidget (object_manager);
+
+        var state = settings.get_boolean ("bluetooth-enabled");
+        update_tooltip (state, false);
+
+        object_manager.global_state_changed.connect ((state, paired) => {
+            update_tooltip (state, paired);
+        });
```

Comment:
Why was this moved to the constructor instead of staying in `get_display_widget ()`?

Reply:
I think this was my requested change though I'm not sure I remember why but this was what I wrote

> You'll need to call update_tooltip () here to initialise your tooltip. Otherwise, the user will only see a tooltip when they pair a device.

I'll check back on this in a bit.

Reply:
Yeah I'm sorry, this was my bad. I'm not sure why I requested for it to be moved out of the constructor. I think what I wanted was just for `update_tooltip ()` to be called after the DisplayWidget is constructed but I'm not sure how my confusion led me to this. Sorry about this, I'll push a commit to revert this.

Reply:
No wait, I didn't request for it to be moved out, just that `update_tooltip ()` be called after construction of the `DisplayWidget` so that the tooltip shows up instead of only showing after a device is paired.

Please ignore my comments in this conversation.